### PR TITLE
Add Project.toml File

### PR DESCRIPTION
**Pull request by ccoffrin**
Updates for latest package manager and new tag bot.

### CODE REVIEW

**Comment by mlubin** [comment]
Would it also be a good time to drop support for Julia pre-1.0, drop the REQUIRE file, and update CI to run only on 1.0?

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+authors = ["Miles Lubin"]
+repo = "https://github.com/JuliaOpt/ConicBenchmarkUtilities.jl.git"
+version = "0.3.2"
+
+[deps]
+GZip = "92fee26a-97fe-5a0c-ad85-20a5f3185b63"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+
+[compat]
```
Comment (mlubin) [inline Project.toml:12]: The tag PR isn't automatically mergeable without version bounds on all dependencies, e.g., `GZip`.

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+JuMP = "~0.18"
+julia = "~0.6, ~0.7, ~1"
```
Comment (mlubin) [inline Project.toml]: This could be `julia = "1"`. Claiming support for pre-1.0 versions has no effect.

**Comment by ccoffrin** [comment]
> Would it also be a good time to drop support for Julia pre-1.0, drop the REQUIRE file, and update CI to run only on 1.0?

That's fine by me and I can do the same on the other legacy projects.  I don't know how to do the maintenance on `appveyor.yml` however.

**Comment by mlubin** [comment]
It should be sufficient to copy the config from MOI: https://github.com/JuliaOpt/MathOptInterface.jl/blob/master/appveyor.yml

**Comment by ccoffrin** [comment]
Ok, I'll work these changes into all the others as well.

**Pull request merged**